free-3.md

```diff
@@ -4,51 +4,96 @@
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would either write a strategy or read a strategy or search for a strategy.
+The reason this could be the right way to interact with the problem domain is that
+it could provide a simplified manner to undestand strategies. Currently in many
+games people will share their thoughts on streams like [Twith](http://www.twitch.tv)
+but there is not an organized collected place to view these strategies.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The program running would be people reading or searching for strategies.  An error
+that could occur is that someone could write a very ineffective strategy and then
+the user could be learning total garbage that would not help there game.  Sadly,
+the only way I can think to communicate this to the user is that they would lose
+a lot of games.  Perhaps if the strategies are posted somewhere there could be a
+field describing the effectiveness of the strategy and then the user could filter out
+ineffective strategies.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to find, write, and read strategies.  It should be impossible
+to automate strategies. The intent of the langauge is to help humans understand
+the strategy not to help create bots that can play the strategy (although this
+DSL would provide a great format for someone who wants to create a bot to get
+ideas about how to program their bot).
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are various forums and such that people post strategies on. There are also
+books written about many more classic games.  I don't believe any of these classify
+as DSL because they have no real syntactic structure (well books have English but
+that's not a DSL).  I think this could be an improvement by providing some universal
+standards across gamees for sharing infomration.
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
+This could be a good CS 111 project since it would require a lot of design to
+make possible.  I think it is substantial enough that it would take a non trivial
+amount of time to develop, but also not so difficult to implemenet that it would
+take too much time.  I think we could finish it by the end of the semester.
```

Comment:
Yes, this sounds like quite a large, interesting project. I think it is definitely something doable if it is focused on one game or one family of games.